enhancements/proposals/1311-replace-nodes-using-hostid/README.md

```diff
@@ -0,0 +1,108 @@
+# Replace ScyllaCluster nodes using Host ID
+
+## Summary
+
+The proposed change aims to improve the replacement procedure of ScyllaCluster nodes by adding new replacement procedure
+and prepare ground for getting out of old one using deprecated ScyllaDB parameters. Both procedures will coexist in
+Scylla Operator, users will have time to shift to a new one until the old one is removed in one of the following Scylla
+Operator versions.
+
+## Motivation
+
+The motivation behind the proposed change is to have a replacement procedure that uses stable node identifiers.
+Currently, the Scylla Operator uses IP addresses of a node to identify which node should be replaced. The issue is that
+depending on the ScyllaCluster configuration, IP addresses might be ephemeral so it’s error prone to use it as an
+identifier as they may change during when the procedure is executed. Because there is a better stable alternative, IP
+address approach became deprecated, hence we need to shift to it.
+
+### Goals
+
+Add replace node procedure using stable Host IDs.
+Remain backward compatible.
+
+### Non-Goals
+
+Remove existing replace procedure.
+
+## Proposal
+
+I propose to add a new node replacement procedure which is using stable Host ID identifiers. Both old and new will
+coexist in Scylla Operator making it safer to implement, allow changes to be rolled out iteratively and without
+compatibility issues during Operator upgrades/downgrades. To make users aware that old procedure is being deprecated, it
+will be visible in release notes as well as a special Warning log message will be printed out in Operator logs telling
+that user is using deprecated procedure and they should shift to the new one.
+
+### Risks and Mitigations
+
+#### ScyllaDB version requirement
+
+Node replacement procedure alignment requires the newest 5.2 ScyllaDB Open Source and 2023.1 Enterprise, meaning node
+replacement procedure won’t work when Scylla doesn’t meet minimum version criteria. Users wanting to update to a newer
+Operator version will need to update their ScyllaDB versions. We will mention it in release notes.
```

Comment:
let's make it also clear that the older scylla version will fallback to the old procedure until it get removed and scylla version requirement raised.

Reply:
done